src/renderer/component/common/icon.jsx

```diff
@@ -1,10 +1,10 @@
 // @flow
 import React from 'react';
-// import * as icons from 'constants/icons';
 import * as FeatherIcons from 'react-feather';
 import * as icons from 'constants/icons';
 
 const RED_COLOR = '#e2495e';
```

Comment:
Would be great to have a common color palette shared that's paired with a style guide.

Reply:
I agree. Currently just copying them from our scss variables file. The values should probably live somewhere outside the app.